components/list/list-item-drag-drop-mixin.js

```diff
@@ -1,23 +1,27 @@
 import { css, html } from 'lit-element/lit-element.js';
+import { findComposedAncestor, isComposedAncestor } from '../../helpers/dom.js';
 import { announce } from '../../helpers/announce.js';
 import { classMap } from 'lit-html/directives/class-map.js';
 import { dragActions } from './list-item-drag-handle.js';
-import { findComposedAncestor } from '../../helpers/dom.js';
 import { getUniqueId } from '../../helpers/uniqueId.js';
 import { ifDefined } from 'lit-html/directives/if-defined.js';
 import { nothing } from 'lit-html';
+import { SelectionInfo } from '../selection/selection-mixin.js';
 
-export const dropLocation = Object.freeze({
+export const moveLocations = Object.freeze({
```

Comment:
I'd like to transition to "move" verbiage for this, to be a bit more semantically correct.